src/client/src/orangehrmRecruitmentPlugin/pages/EditJobVacancy.vue

```diff
@@ -473,11 +473,13 @@ export default {
         this.vacancy.numOfPositions = data.numOfPositions || '';
         this.vacancy.status = data.status;
         this.vacancy.isPublished = data.isPublished;
-        this.vacancy.hiringManager = {
-          id: data.hiringManager.id,
-          label: `${data.hiringManager.firstName} ${data.hiringManager.middleName} ${data.hiringManager.lastName}`,
-          isPastEmployee: data.hiringManager.terminationId ? true : false,
-        };
+        this.vacancy.hiringManager = data.hiringManager.id
+          ? {
+              id: data.hiringManager.id,
+              label: `${data.hiringManager.firstName} ${data.hiringManager.middleName} ${data.hiringManager.lastName}`,
+              isPastEmployee: data.hiringManager.terminationId ? true : false,
+            }
+          : this.$t('general.deleted');
```

Comment:
Attach screenshots for both scenarios. with deleted and with not deleted

Reply:
![Screenshot from 2023-06-28 11-39-20](https://github.com/orangehrm/orangehrm/assets/49731724/23c9687e-357e-4c20-bb08-5a3c8d89cba7)
![Screenshot from 2023-06-28 11-38-08](https://github.com/orangehrm/orangehrm/assets/49731724/78f7abc6-c62c-42e6-b3a4-99f478c47192)